src/reference/configuration/listening_address.md

```diff
@@ -30,3 +30,44 @@ Open the Connection Manager and proceed with the following steps:
 
 2. Select the appropriate radio button. _If using a_ `Custom` _listening address - supply the address in the input field._
 3. Supply the value of the desired listening port.
+
+## Proxying Local Traffic
+
+If you are having issues proxying localhost traffic - try the following:
+
+#### Use lvh.me:
+
+This domain name resolves to 127.0.0.1.
+
+- [http://lvh.me/](http://lvh.me/)
+
+#### Edit the proxy bypass rules:
+
+By default, localhost may be included as a proxy bypass address within the list of hosts used by your system/browser/extension. This can be overwritten by supplying `<-loopback>` (_the `-` character inverts the rule_).
+
+- Launch Chrome via the terminal using:
+
+```
+google-chrome --proxy-server=127.0.0.1:8080` --proxy-bypass-list="<-loopback>"
+```
+
+::: tip
+If you customized the listening address/port of Caido - replace the values in the command to match. Click [here](https://chromium.googlesource.com/chromium/src/+/HEAD/net/docs/proxy.md#Overriding-the-implicit-bypass-rules) for more information.
+:::
+
+- If you are using the FoxyProxy browser extension: select `Options` and add `<-loopback>` to the `Global Exclude` list.
+- In general, this bypass list can be modified in the proxy settings of different systems/browsers/extensions by supplying `<-loopback>` to the hosts list. This input field is usually accompanied with a title or description that includes key-terms/keywords such as: `except these addresses`, `no-proxy for`, `exclude`, etc.
+
+#### Use a local DNS record:
+
+- [https://www.tecmint.com/setup-local-dns-using-etc-hosts-file-in-linux/](https://www.tecmint.com/setup-local-dns-using-etc-hosts-file-in-linux/)
```

Comment:
The tecmint article was just the first I found on the topic. I am not sure it is a reputable source, its also unix specific so we should mention that.

Reply:
Just going to exclude this option. The others work just fine. Adding a host entry that is linked to 127.0.0.1 does work but only with FoxyProxy on. I cannot edit the proxy settings on my Kali VM.